app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.health = 100
+        self.hidden = False
+        self.__class__.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+    def die(self) -> None:
+        self.__class__.alive.remove(self)
```

Comment:
u can just use class name here, to write less code
also u can make this method as classmethod